galley/pkg/config/analysis/diag/message.go

```diff
@@ -21,6 +21,8 @@ import (
 	"istio.io/istio/galley/pkg/config/resource"
 )
 
+const DocPrefix = "https://istio.io/docs/reference/config/analysis"
```

Comment:
If we're going to enforce people adding docs (and we should) then I think we'd better have the ability to also point to preliminary.istio.io as well as the prod site. I don't really have a great idea on how to accomplish this :) But I wanted to note it in case it affects how we think about using this prefix. I don't think it's a blocker for this PR.

Reply:
As discussed offline, this is less of an issue now that we've backed away from encouraging the use of tip-of-master.